app/Helpers/MailSend.php

```diff
@@ -16,9 +16,10 @@ class MailSend
      * @param $data
      * @param $profile
      * @param $subject
+     * @param null $pdf
      * @return bool
      */
-    public static function send($view, $data, $profile, $subject)
+    public static function send($view, $data, $profile, $subject, $pdf = null)
```

Comment:
make `$pdf` to be array of `$attachments` and iterate them.

`MailSend` shouldn't be aware of what the attachment is.